test/vaadin-combo-box-light.html

```diff
@@ -142,6 +151,73 @@
       });
     });
 
+    describe('vaadin-combobox-light-attr-for-value', () => {
+      let comboBox, customInput, ironInput, nativeInput;
+
+      function whenIronInputReady(ironInput, callback) {
```

Comment:
can we use `ironInput._observer.flush()` instead of this callback?
see the observer to init slotted input detection [here](https://github.com/PolymerElements/iron-input/blob/5f9099a045554e9fe666bca614b22c18f77247e5/iron-input.html#L174)

Reply:
I'll check it out, thanks for the suggestion.

Reply:
Seems to work, and simplifies the code quite a bit. Thanks!

Reply:
Update pushed.